ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -118,6 +120,42 @@ const EthOverview = ({ className, showAddress }) => {
   }
 
   const isSwapsChain = useSelector(getIsSwapsChain);
+  const signingEnabled = useSelector((state) =>
```

Comment:
I'm gonna be picky again, so feel free to ignore this one 🙈

We could rename it to `isSigningEnabled`:
1. That would match the style than other boolean values (cosmetic)
2. It complies (IMO) better with the following rule: https://github.com/MetaMask/contributor-docs/blob/main/docs/javascript.md#use-a-verb-phrase-for-boolean-variable-names-that-do-not-involve-secondary-objects

WDYT?

Reply:
Very good point! I renamed it to `isSigningEnabled` here [75089aa](https://github.com/MetaMask/metamask-extension/commit/75089aa4d7eace3ce29c152b70e2a90bc3bf8891)